api/api.gradle.kts

```diff
@@ -13,6 +13,7 @@ dependencies {
     api(group = "de.codecentric.centerdevice", name = "javafxsvg", version = "1.2.1")
     api(group = "edu.wpi.first.ntcore", name = "ntcore-java", version = "4.+")
     api(group = "eu.hansolo", name = "Medusa", version = "7.9") // Note the capital 'M' -- lowercase is a much older version!
+    api(group = "com.cedarsoft.commons", name = "version", version = "8.3.1")
```

Comment:
Curious why you picked this instead of this other library.
https://github.com/zafarkhaja/jsemver

Did you see both and pick this one because it had better support?

Reply:
The cedarsoft one is smaller, easier to understand, and has built in support for checking stuff like `versionA >= versionB`. Also has a `VersionRange` class that might be useful for bounded dependency versions